src/main/java/org/opensearch/knn/index/query/parser/MethodParametersParser.java

```diff
@@ -29,7 +29,6 @@
 import java.util.Map;
 import java.util.function.Function;
 
-import static org.opensearch.knn.index.IndexUtil.isClusterOnOrAfterMinRequiredVersion;
```

Comment:
Why are we removing this ?

Reply:
It was a leftover from this PR https://github.com/opensearch-project/k-NN/pull/1742/files. The import is not needed. Sorry for the confusion